identityClient/identityClient.go

```diff
@@ -562,6 +563,50 @@ func (c *E3dbIdentityClient) GetToznyHostedBrokerInfo(ctx context.Context) (*Toz
 	return toznyHostedBrokerInfo, err
 }
 
+// InitiateIdentityLogin begins the standard 3rd party login flow with TozID
+func (c *E3dbIdentityClient) InitiateIdentityLogin(ctx context.Context, loginRequest IdentityLoginRequest) (*map[string]interface{}, error) {
+	var resp *map[string]interface{}
```

Comment:
Does this use `InitialLoginResponse`

Reply:
It is supposed to.  @lkwdwrd in other sdks we have just treated this as a map that gets passed back. Is that because we can't guarantee its shape or is this safe to strongly type?

Reply:
I am afraid I don't recall specifically -- It was most likely a 'lets get this working and come back and add the type once the requirements are fully understood' situation, but the type was never added. We should define the type for it.